VERSION ""


NS_ :
    NS_DESC_
    CM_
    BA_DEF_
    BA_
    VAL_
    CAT_DEF_
    CAT_
    FILTER
    BA_DEF_DEF_
    EV_DATA_
    ENVVAR_DATA_
    SGTYPE_
    SGTYPE_VAL_
    BA_DEF_SGTYPE_
    BA_SGTYPE_
    SIG_TYPE_REF_
    VAL_TABLE_
    SIG_GROUP_
    SIG_VALTYPE_
    SIGTYPE_VALTYPE_
    BO_TX_BU_
    BA_DEF_REL_
    BA_REL_
    BA_DEF_DEF_REL_
    BU_SG_REL_
    BU_EV_REL_
    BU_BO_REL_
    SG_MUL_VAL_

BS_:

BU_: XXX


BO_ 276 NEW_MSG_1: 8 XXX

BO_ 292 ENGINE_MEAS: 8 XXX
 SG_ RPM : 15|16@0+ (1,0) [0|6000] "" XXX
 SG_ NEW_SIGNAL_1 : 47|16@0- (1,0) [0|15] "" XXX

BO_ 288 ENGINE_1: 6 XXX
 SG_ NEW_SIGNAL_1 : 11|12@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_2 : 27|12@0+ (1,0) [0|15] "" XXX

BO_ 290 ENGINE_CONTROL: 5 XXX
 SG_ GAS_1 : 15|8@0+ (1,0) [0|255] "" XXX
 SG_ GAS_2 : 23|8@0+ (1,0) [0|255] "" XXX
 SG_ OIL_PRES : 24|16@1+ (1,0) [0|255] "" XXX

BO_ 440 WHEEL_SPEEDS: 8 XXX
 SG_ WHEEL_FRONT_LEFT : 23|16@0+ (1,0) [0|511] "" XXX
 SG_ WHEEL_FRONT_RIGHT : 7|16@0+ (1,0) [0|65535] "" XXX
 SG_ WHEEL_REAR_LEFT : 55|16@0+ (1,0) [0|65535] "" XXX
 SG_ WHEEL_REAR_RIGHT : 39|16@0+ (1,0) [0|65535] "" XXX

BO_ 319 ENGINE_RPM: 8 XXX
 SG_ rpm : 13|14@0+ (0.23,0) [0|6000] "/min" XXX

BO_ 495 STEER_NOISY: 8 XXX
 SG_ STEER_NOISY_1 : 15|16@0+ (1,0) [0|65535] "" XXX
 SG_ STEER_NOISY_2 : 31|16@0+ (1,0) [0|255] "" XXX
 SG_ LONG : 46|10@0+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_1 : 63|8@0- (1,0) [0|255] "" XXX

BO_ 672 NEW_MSG_8: 8 XXX
 SG_ NEW_SIGNAL_2 : 23|16@0+ (1,0) [0|65535] "" XXX
 SG_ NEW_SIGNAL_3 : 39|16@0+ (1,0) [0|65535] "" XXX
 SG_ NEW_SIGNAL_4 : 55|16@0+ (1,0) [0|65535] "" XXX
 SG_ NEW_SIGNAL_1 : 7|16@0+ (1,0) [0|255] "" XXX

BO_ 788 LONGITUDINAL: 8 XXX
 SG_ SAW : 31|16@0+ (1,0) [0|65535] "" XXX
 SG_ SPEED : 15|16@0+ (1,0) [0|65535] "" XXX
 SG_ GAS_PROCESSED : 47|16@0+ (1,0) [0|65535] "" XXX

BO_ 953 NEW_MSG_10: 8 XXX
 SG_ NEW_SIGNAL_2 : 39|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_1 : 15|8@0+ (1,0) [0|255] "" XXX
 SG_ NEW_SIGNAL_3 : 31|8@0+ (1,0) [0|255] "" XXX

BO_ 431 NEW_MSG_7: 8 XXX

BO_ 976 INDICATORS: 8 XXX
 SG_ BLINKER_LEFT : 38|1@0+ (1,0) [0|1] "" XXX
 SG_ BLINKER_RIGHT : 39|1@0+ (1,0) [0|1] "" XXX
 SG_ PARKING_BRAKE : 14|1@0+ (1,0) [0|1] "" XXX

BO_ 977 NEW_MSG_11: 8 XXX
 SG_ DOOR_LEFT : 0|1@1+ (1,0) [0|3] "" XXX
 SG_ NEW_SIGNAL_1 : 55|16@0+ (1,0) [0|255] "" XXX

BO_ 792 NEW_MSG_12: 8 XXX
 SG_ NEW_SIGNAL_1 : 39|8@0- (1,0) [0|65535] "" XXX

BO_ 429 ACCEL: 8 XXX
 SG_ LATERAL : 35|8@0+ (1,0) [0|15] "" XXX

BO_ 281 STEER_INPUT: 8 XXX
 SG_ STEER_SMOOTH_2 : 23|16@0+ (1,0) [0|65535] "" XXX
 SG_ STEER_SMOOTH_1 : 39|16@0+ (1,0) [0|65535] "" XXX
 SG_ NEW_SIGNAL_1 : 13|6@0+ (1,0) [0|63] "" XXX

BO_ 784 NEW_MSG_15: 8 XXX
 SG_ NEW_SIGNAL_1 : 15|8@0+ (1,0) [0|255] "" XXX

BO_ 1280 NEW_MSG_16: 8 XXX

BO_ 1281 NEW_MSG_17: 8 XXX

BO_ 1000 NEW_MSG_6: 8 XXX
 SG_ NEW_SIGNAL_1 : 15|8@0+ (1,0) [0|255] "" XXX
